docs/cmdlets/Trace-PipelineObject.md

```diff
@@ -13,7 +13,7 @@ Writes the formatted output of the pipeline object to the information stream bef
 ## SYNTAX
 
 ```
-Trace-PipelineObject [-InputObject] <Object> [<CommonParameters>]
+Trace-PipelineObject -InputObject <Object> [<CommonParameters>]

```

Comment:
Can me go back to this being positional?

Reply:
Yes, will make the change.

Reply:
Fixed.